### PR TITLE
Allow some symbolic constants in place of dissolve delay seconds

### LINKED ISSUE #24: "Months" have weak meaning when it comes to dissolve delays
In all of our literature we talk about "6 months" being the minimum dissolve delay that can accrue voting rewards, except that we use seconds in the `quill` user interface and don't specify which six months, in which year, are being used to measure those seconds.

We should both document that 15_778_800 seconds is the actual minimum number of seconds required to vote, and also provide a constant string such as SIX_MONTHS that can be passed in the place of a seconds figure, to make it easier for users. Better still would be to add a simple expression parser that handles basic arithmetic, and then make all the same constants we use in Rust available, e.g.:

```
quill neuron-manage --additional-dissolve-delay-seconds '(8 * MONTHS) + 2 YEARS'
```

### PR DESCRIPTION
**Pull request by jwiegley**
fixes #24

### CODE REVIEW

File: src/commands/neuron_manage.rs [hunk 1]
```diff
@@ -185,7 +190,39 @@ pub async fn exec(
...
+                        "ONE_YEAR" => ONE_YEAR_SECONDS * 12,
+                        "TWO_YEARS" => ONE_YEAR_SECONDS * 12 * 2,
+                        "THREE_YEARS" => ONE_YEAR_SECONDS * 12 * 3,
+                        "FOUR_YEARS" => ONE_YEAR_SECONDS * 12 * 4,
+                        "FIVE_YEARS" => ONE_YEAR_SECONDS * 12 * 5,
+                        "SIX_YEARS" => ONE_YEAR_SECONDS * 12 * 6,
+                        "SEVEN_YEARS" => ONE_YEAR_SECONDS * 12 * 7,
+                        "EIGHT_YEARS" => ONE_YEAR_SECONDS * 12 * 8,
+
+                        s => s.parse::<u32>().unwrap(),
```
Comment (chmllr) [inline src/commands/neuron_manage.rs:224]:
```suggestion
                        s => s.parse::<u32>().expect("Couldn't parse the dissolve delay"),
```

File: src/commands/neuron_manage.rs [hunk 2]
```diff
@@ -185,7 +190,39 @@ pub async fn exec(
...
+                        "TWO_WEEKS" => ONE_DAY_SECONDS * 7 * 2,
+                        "THREE_WEEKS" => ONE_DAY_SECONDS * 7 * 3,
+                        "FOUR_WEEKS" => ONE_DAY_SECONDS * 7 * 4,
+
+                        "ONE_MONTH" => ONE_MONTH_SECONDS,
+                        "TWO_MONTHS" => ONE_MONTH_SECONDS * 2,
+                        "THREE_MONTHS" => ONE_MONTH_SECONDS * 3,
+                        "FOUR_MONTHS" => ONE_MONTH_SECONDS * 4,
+                        "FIVE_MONTHS" => ONE_MONTH_SECONDS * 5,
+                        "SIX_MONTHS" => ONE_MONTH_SECONDS * 6,
```
Comment (chmllr) [inline src/commands/neuron_manage.rs:208]: Btw, does this correspond to the minimal amount of seconds to get the rewards?

Comment (jwiegley) [reply src/commands/neuron_manage.rs:208]: Yes, exactly.

**Review by chmllr — APPROVED** [review]

**Comment by jwiegley** [comment]
I'll document these in the README when I'm making the README changes for the Nix stuff.

**Pull request merged**